revolv/templates/donor/dashboard.html

```diff
@@ -3,79 +3,129 @@
+              <div class="overlay"></div>
+              <div class="status-bar">
+                <a href="#"><p class="status-bar-menu-text">{{project.title}}</p></a>
+              </div>
+            </div>
+          {% endfor %}
+          {% for project in drafted_projects %}
+            <div class="project-container large-4 columns">
+              <img src='{{project.cover_photo.url}}'/>
+              <div class="overlay"></div>
+              <div class="status-bar">
+                <a href="#"><p class="status-bar-menu-text">{{project.title}}</p></a>
+              </div>
+            </div>
+          {% endfor %}
+          {% for project in completed_projects %}
+            <div class="project-container large-4 columns">
+              <img src='{{project.cover_photo.url}}'/>
+              <div class="overlay"></div>
+              <div class="status-bar">
+                <a href="#"><p class="status-bar-menu-text">{{project.title}}</p></a>
+              </div>
+            </div>
+          {% endfor %}
+         </div></div>
+         <div class="large-3 pull-9 columns>">
+          <br></br>
+          <div class="large-10 pull--1 columns>">
+          <ul class="accordion" data-accordion>
+            <li class="accordion-navigation">
+              <a href="#panel1">Active Projects</a>
+              <div id="panel1" class="content">
+                <ul class="side-nav">
+                  {% for project in active_projects %}
+                    <li><a href="project/{{project.id}}">{{project.title}}</a></li>
+                  {% endfor %}
+                </ul>
+              </div>
+            </li>
+            <li class="accordion-navigation">
+              <a href="#panel2">Proposed Projects</a>
+              <div id="panel2" class="content">
+                <ul class="side-nav">
+                  {% for project in proposed_projects %}
+                    <li><a href="project/{{project.id}}">{{project.title}}</a></li>
+                  {% endfor %}
+                </ul>
+              </div>
+            </li>
+            <li class="accordion-navigation">
+              <a href="#panel3">Drafted Projects</a>
+              <div id="panel3" class="content">
+                <ul class="side-nav">
+                  {% for project in drafted_projects %}
+                    <li><a href="project/{{project.id}}">{{project.title}}</a></li>
+                  {% endfor %}
+                </ul>
+              </div>
+            </li>
+            <li class="accordion-navigation">
+              <a href="#panel4">Completed Projects</a>
+              <div id="panel4" class="content">
+                <ul class="side-nav">
+                  {% for project in completed_projects %}
+                    <li><a href="project/{{project.id}}">{{project.title}}</a></li>
+                  {% endfor %}
+                </ul>
+              </div>
+            </li>
```

Comment:
I think you can collapse this by writing a `for` loop:
`{% for type in ['Active', 'Proposed', 'Drafted', 'Completed'] %}`